ses

```diff
@@ -175,14 +175,21 @@ else
     plugin_message "neither /usr/bin/ceph nor /usr/sbin/cephadm found"
 fi
 
-if [ -x /usr/sbin/ceph-disk ]; then
-    plugin_command "ceph-disk list" > $LOG/ceph/ceph-disk-list 2>&1
+if [ -x /usr/sbin/cephadm ] ; then
+    # `cephadm ceph-volume` dumps ceph-volume output to stderr prefixed with
+    # INFO:cephadm:/usr/bin/podman:stdout, so grepping that junk out to avoid
+    # duplicating the output.
+    plugin_command "cephadm ceph-volume lvm list" 2>&1 | grep -v '/usr/bin/podman:stdout' > $LOG/ceph/ceph-volume-list
+    plugin_command "cephadm ceph-volume inventory -- --format json-pretty" 2>&1 | grep -v '/usr/bin/podman:stdout' > $LOG/ceph/ceph-volume-inventory.json
+    plugin_command "cephadm ceph-volume inventory" 2>&1 | grep -v '/usr/bin/podman:stdout' > $LOG/ceph/ceph-volume-inventory
 elif [ -x /usr/sbin/ceph-volume ]; then
     plugin_command "ceph-volume lvm list" > $LOG/ceph/ceph-volume-list 2>&1
     plugin_command "ceph-volume inventory --format json-pretty" > $LOG/ceph/ceph-volume-inventory.json 2>&1
     plugin_command "ceph-volume inventory" > $LOG/ceph/ceph-volume-inventory 2>&1
+elif [ -x /usr/sbin/ceph-disk ]; then
+    plugin_command "ceph-disk list" > $LOG/ceph/ceph-disk-list 2>&1
```

Comment:
As I mentioned, no idea how the versioning works, but might be worth dropping `ceph-disk` here? mimic was the last release to ship `ceph-disk`